src/config.c

```diff
@@ -688,6 +719,63 @@ void configSetCommand(redisClient *c) {
 
         if (yn == -1) goto badfmt;
         server.aof_rewrite_incremental_fsync = yn;
+#ifdef USE_NVML
+    } else if (!strcasecmp(c->argv[2]->ptr, "aof-use-nvml")){
+        int enable = yesnotoi(o->ptr);
+
+        if (enable == -1) goto badfmt;
+
+        if (server.aof_use_nvml != enable) {
+            if (server.aof_state == REDIS_AOF_OFF) {
+                server.aof_use_nvml = enable;
+            } else if (server.aof_state == REDIS_AOF_ON) {
+                // - change aof-nvml-value
+                // - rewrite aof
+                server.aof_use_nvml = enable;
+                redisLog(REDIS_NOTICE,"Restarting AOF to switch NVML mode");
+                stopAppendOnly();
+                if (startAppendOnly() == REDIS_ERR) {
```

Comment:
So, at this point, the old (classic) AOF is lost?
I don't think this option could be configurable at run-time. At least, not if AOF is enabled.